packages/sampler/stories/default/ViewManager.js

```diff
@@ -0,0 +1,135 @@
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, select} from '@enact/storybook-utils/addons/knobs';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import Group from '@enact/ui/Group';
+import Item from '@enact/ui/Item';
+import Layout, {Cell} from '@enact/ui/Layout';
+import Scroller from '@enact/ui/Scroller';
+import {arrange, SlideArranger, SlideBottomArranger, SlideLeftArranger, SlideRightArranger, SlideTopArranger, ViewManager, ViewManagerBase} from '@enact/ui/ViewManager';
+import {useCallback, useState} from 'react';
+
+import css from './ViewManager.module.less';
+
+const ViewManagerConfig = mergeComponentMetadata('ViewManager', ViewManagerBase, ViewManager);
+
+ViewManagerConfig.defaultProps = {
+	duration: 300,
+	enteringDelay: 0
+};
+
+const prop = {
+	arranger: ['SlideBottomArranger', 'SlideLeftArranger', 'SlideRightArranger', 'SlideTopArranger', 'SlideArranger', 'CustomArranger (FadeAndSlideArranger)'],
+	direction: ['bottom', 'left', 'right', 'top']
+};
+
+const arrangers = {
+	'SlideBottomArranger': SlideBottomArranger,
+	'SlideLeftArranger': SlideLeftArranger,
+	'SlideRightArranger': SlideRightArranger,
+	'SlideTopArranger': SlideTopArranger
+};
+
+const views = new Array(10).fill().map((i, index) => {
+	return {
+		title: `Item ${index + 1}`,
+		content: `This is View ${index + 1}`
+	};
+});
+
+const ViewManagerLayout = (props) => {
+	const [selected, setSelected] = useState(0);
+	const handleChangeView = useCallback((ev) => {
+		setSelected(ev.selected);
+	}, [setSelected]);
+
+	return (
+		<Layout>
+			<Cell component={Scroller} size="7%">
+				<Group
+					className={css.group}
+					childComponent={Item}
+					itemProps={{className: css.navItem}}
+					onSelect={handleChangeView}
+					select="radio"
+				>
+					{views.map((view) => view.title)}
+				</Group>
+			</Cell>
+			<Cell
+				className={css.viewport}
+				component={ViewManager}
+				index={selected}
+				{...props}
+			>
+				{views.map((view, i) => (
+					<div className={css.box} key={i}>
+						{view.content}
+					</div>
+				))}
+			</Cell>
+		</Layout>
+	);
+};
+
+export default {
+	title: 'UI/ViewManager',
+	component: 'ViewManager'
+};
+
+export const _ViewManager = () => {
+	const arrangerType = select('arranger', prop.arranger, ViewManagerConfig, prop.arranger[0]);
+	let arranger;
+
+	if (arrangerType === 'SlideArranger') {
+		const amount = number('amount', ViewManagerConfig, {range: true, min: 0, max: 100}, 100);
+		const direction = select('direction', prop.direction, ViewManagerConfig, prop.direction[0]);
```

Comment:
These two prop is not ViewManager prop. and it make confusion.
Let's make another knob tab 

```suggestion
		const amount = number('amount', SliderArrangerConfig, {range: true, min: 0, max: 100}, 100);
		const direction = select('direction', prop.direction, SliderArrangerConfig, prop.direction[0]);
```

Reply:
OK